tsconfig.json

```diff
@@ -20,6 +20,7 @@
 		"./build",
 		"./spec",
 		"./demo",
+		"./examples",
```

Comment:
Another exclude? Might be better to only include "src" instead?
```json
{
    ...
    "include": [
        "./src"
    ]
}
```
_(I can not suggest the changes 😕)_